plugins/woocommerce/package.json

```diff
@@ -239,6 +240,20 @@
 					"testEnv": {
 						"start": "env:perf"
 					}
+				},
+				{
+					"name": "Metrics",
+					"testType": "performance",
+					"command": "test:metrics:ci",
+					"changes": [
+						"client/admin/config/*.json",
+						"composer.lock",
+						"includes/**/*.php",
+						"patterns/**/*.php",
+						"src/**/*.php",
+						"templates/**/*.php",
```

Comment:
Should we include the blocks templates: `templates/**/*.html`?

Reply:
That makes sense. Included with https://github.com/woocommerce/woocommerce/pull/46471/commits/46769cab3c65bd62d664fe3775057c287c15f576